skyportal/handlers/api/followup_request.py

```diff
@@ -412,9 +437,20 @@ def get(self, followup_request_id=None):
             joinedload(FollowupRequest.allocation).joinedload(Allocation.group),
             joinedload(FollowupRequest.obj),
             joinedload(FollowupRequest.requester),
-        ).all()
+        )
+
+        total_matches = followup_requests.count()
```

Comment:
Should there be a hard maximum number of requests? Like for candidates?

Reply:
But just counting is ok?

Reply:
I mean that if the user tries to input 10,000 as the number per page (or set `None`) it should not work.

Reply:
Set it to 1000 for now.